packages/search-ui/src/__tests__/URLManager.test.ts

```diff
@@ -1,19 +1,12 @@
+import { SearchState } from "../types";
 import URLManager from "../URLManager";
 
 function createManager() {
   const manager = new URLManager();
-  manager.history = {
```

Comment:
I changed URLManager to detect what environment its running in (server or browser) and to use the memoryHistory implementation if running on server. In tests we are therefore using the memoryHistory and therefore no need to provide a mock